src/query/src/attribute.ts

```diff
@@ -5,6 +5,11 @@ import { asAttributeNode, type ParserState } from './types.js'
 
 export type ComparatorFn = (attr: string, value?: string) => boolean
 
+// JSONField has a mapped type constituent that would coerce to [object Object]
+// when stringified, which is what we want in this case.
```

Comment:
@ruyadorno Is this comment I added accurate? There is a test case that asserts we convert an object to the string `"[object Object]"`, so I wanted to check that this is not a bug.

Reply:
yes 👍 that LGTM, the idea is that we want a direct match to the attribute provided in the user query, so an object being converted to `"[object Object]"` since it won't match user input